src/Spec-Core.package/OkToolbar.class/instance/addAll.withSpec..st

```diff
@@ -3,4 +3,4 @@ addAll: aWindow  withSpec: aSpec
 
 	aWindow
 		addMorph: (self buildWithSpec: aSpec)
-		fullFrame: ((0@0 corner: 1@0) asLayoutFrame bottomOffset: 30).
\ No newline at end of file
+		fullFrame: ((0@0 corner: 1@0) asSpecLayoutFrame bottomOffset: 30).
```

Comment:
Spec used LayoutFrame a lot but SpecLayoutFrame already existed. Code was adopted to use this class.